requirements.txt

```diff
@@ -0,0 +1,12 @@
+numpy
+scipy
+rasterio
+scikit-image
+pytest
```

Comment:
Could be split off in requirements-dev.txt to avoid too many dependencies with regular installation.